src/odemis/acq/test/millmng_test.py

```diff
@@ -71,6 +73,8 @@ class MillingManagerTestCase(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
+        cls.millings = load_config(MILLING_CONFIG_PATH + "/milling-series-test-working-config.mill.yaml")
+
```

Comment:
Please move this call to each function that test `load_config()`. We typically don't want setUpClass to test the code.